content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -272,16 +275,72 @@ transaction functions. We recommend you get in and out as quick as possible!
+Prisma does not currently support this feature on MongoDB.
+
+</Admonition>
+
+From version 4.2.0, you can set the transaction [isolation level](https://www.prisma.io/dataguide/intro/database-glossary#isolation-levels).
+
+<Admonition type="info">
+
+**Before version 4.2.0**<br /><br />
+In version 4.1.0 and earlier, you cannot configure the transaction isolation level
+at a Prisma level. Also, the isolation level is not explicitly set by Prisma, so
+the isolation level is set at the database level, at the default level for the database
+type. For information on default isolation levels, see the following resources:<br /><br />
 
 - [Default transaction isolation level in PostgreSQL](https://www.postgresql.org/docs/9.3/runtime-config-client.html#GUC-DEFAULT-TRANSACTION-ISOLATION)
 - [Default transaction isolation level in Microsoft SQL Server](https://docs.microsoft.com/en-us/sql/t-sql/statements/set-transaction-isolation-level-transact-sql?view=sql-server-ver15)
 - [Default transaction isolation level in MySQL](https://dev.mysql.com/doc/refman/8.0/en/innodb-transaction-isolation-levels.html)
 
-## Join the Conversation on GitHub
+</Admonition>
+
+To set the transaction isolation level, use the `isolation_level` option in the second parameter of the API:
+
+For example:
+
+```jsx
+await prisma.$transaction(
+  async (prisma) => {
+    // Code running in a transaction...
+  },
+  {
+    isolationLevel: Prisma.TransactionIsolationLevel.Serializable, // optional
+    maxWait: 5000, // default: 2000
+    timeout: 10000, // default: 5000
+  }
+)
+```
 
-If you'd like to see transactions supported in the future, [please join the discussion on GitHub](https://github.com/prisma/prisma-client-js/issues/349).
+Prisma Client supports the following isolation levels if they are available in the underlying database:
+
+- `ReadCommitted`
+- `ReadUncommitted`
+- `RepeatableRead`
+- `Serializable`
+- `Snapshot`
+
+##### Isolation levels by database
+
+The isolation levels available for each database connector are as follows:
+
+| Database    | `ReadCommitted` | `ReadUncommitted` | `RepeatableRead` | `Serializable` | `Snapshot` |
+| ----------- | --------------- | ----------------- | ---------------- | -------------- | ---------- |
+| PostgreSQL  | ✔️              | ✔️                | ✔️               | ✔️             | No         |
+| MySQL       | ✔️              | ✔️                | ✔️               | ✔️             | No         |
+| SQL Server  | ✔️              | ✔️                | ✔️               | ✔️             | ✔️         |
+| CockroachDB | No              | No                | No               | ✔️             | No         |
+| SQLite      | No              | No                | No               | ✔️             | No         |
+
+The default isolation level in Prisma Client is the same as the default for your database:
+| Database | Default |
```

Comment:
This table might not be correct for all users as default isolation level is usually configurable.  I'd either omit this table completely or point to the same docs for setting up default levels from above.

Reply:
Thanks - I've removed the table and pointed to the same docs as above.